### PR TITLE
Thread sphere 7

### PR DESCRIPTION
**Pull request by ellaycodes**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -167,3 +166,102 @@ describe("GET /api/articles", () => {
...
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("GET: 200 - responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body);
```
Comment (Hard-Luck) [inline __tests__/app.test.js:176]: remove this

File: __tests__/app.test.js [hunk 2]
```diff
@@ -167,3 +166,102 @@ describe("GET /api/articles", () => {
...
+
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(comment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toEqual('You need to write a comment to post');
+      });
+  });
+});
```
Comment (Hard-Luck) [inline __tests__/app.test.js:280]: what if the article_id is not valid? `/articles/banana/comments`

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
We need one more test for post

File: __tests__/app.test.js (hunk 1 above)
Comment (ellaycodes) [reply __tests__/app.test.js:176]: I'm not sure why I need to remove this, could you explain a little more

**Review by johnhutcheon — APPROVED** [review]
Hey Ella, thanks for adding the further test. We say to remove console logs when pushing to github, as we should get in the pattern of only pushing production ready code, with no console logs or notes the developer might have put in for themselves. 👍
Hope that clears it up, you're good to merge! 🙂

**Pull request merged**